src/client/components/share.ts

```diff
@@ -96,7 +97,7 @@ export class Share extends LitElement {
     }) public target?: URL;
 
     // Required.
-    @property({ attribute: 'article-title' }) public articleTitle?: string;
+    @property({ type: String, attribute: 'article-title' }) public articleTitle: string = '';
```

Comment:
These two changes are big part of why you knobs weren't working. Knobs can't represent an `undefined` value, so nothing that defaults to `undefined` can be managed by a knob. 😬